server/eclipse-project/src/test/resources/corlaInit.sql

```diff
@@ -0,0 +1,1928 @@
+--
+-- PostgreSQL database dump
```

Comment:
Just a note on this -- this dump will need to be updated as the new entities are implemented and any existing entities changed (and care taken that it is a dump of the database with just basic test data in it).

Reply:
True. It's probably not too bad because there are only a relatively small number of changes, but yes, anyone who changes the database schema will need to change this file too.

It was very easy to create - just an SQL export using intelliJ's database viewer. I did have to comment out some things at the top about the database user.